scss/mixins/_table-row.scss

```diff
@@ -9,6 +9,15 @@
     > td {
       background-color: $background;
     }
+
+    @if $border != null {
```

Comment:
Why wrap this around a `null` conditional?

Reply:
Probably because `$border` is initialized to `null` above. And this way it's an optional parameter.